django_declarative_apis/machinery/filtering.py

```diff
@@ -195,5 +265,9 @@ def apply_filters_to_object(inst, filter_def, expand_header=""):
     else:
         expand_dict = {}
     return _apply_filters_to_object(
-        inst, filter_def, expand_children=expand_dict, klass=inst.__class__
+        inst,
+        filter_def,
+        expand_children=expand_dict,
+        klass=inst.__class__,
+        filter_cache={},
```

Comment:
How concerned should we be about the size of this object that we're passing? It could be massive, right?

Reply:
Maybe worthwhile doing something like this to track?
```
cache_obj = {}
filtered_val = _apply_filters_to_object(..., filter_cache=cache_obj, ...)
logger.info(... cache_obj.__sizeof__()...)
return filtered_val
```

Reply:
@smholloway it's pass-by-reference, so no concern about size when passing it in function calls.  I don't even really think we need to track it; the total `filter_cache` object is guaranteed to be smaller than the size of the overall response `dict` that we're returning anyway.

Reply:
Alright. Makes sense to me 👍